File: rest/outgoing-caller-ids/instance-post-example-1.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "AC228ba32a3c49700934481addd5ce1659"
auth_token  = "{{ auth_token }}"
client = TwilioRestClient(account_sid, auth_token)

caller_id = client.caller_ids.update("PNe536d32a3c49700934481addd5ce1659", friendly_name="My Second Line")
print caller_id.phone_number

File: rest/call-feedback/instance-post-example-1/instance-post-example-1.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'AC1365ff479ef6502d85c27be6467a310c';
var authToken = "{{ auth_token }}";
var client = require('twilio')(accountSid, authToken);

client.calls("CAe03b7cd806070d1f32bdb7f1046a41c0").feedback.create({
    quality_score: "3",
    issue: "imperfect-audio"
}, function(err, feedback) {
    console.log(feedback.dateCreated);
});
